src/View/Helper/HtmlHelper.php

```diff
@@ -344,7 +344,14 @@ public function link($title, $url = null, array $options = [])
             $title = h(urldecode($title));
             $escapeTitle = false;
         }
-
+        
+        /* A "simple phishing" vulnerability
+         * Consider adding a configuration option to disable?
+         */ https://dev.to/ben/the-targetblank-vulnerability-by-example
+        if (isset($options['target'])) {
+            $options['rel'] = 'noopener noreferrer';
+        }
+        
```

Comment:
The code I shared above was copied from a custom helper where I had this automation implemented ;)

It will only append to the user's defined `rel`.